### ISSUE TITLE
GENIE: decoder_nll loss is always equal to 0 for training from scratch

### ISSUE THREAD

**Issue by BaohaoLiao** [microsoft/ProphetNet#58]
Hi @qiweizhen,

I try to reproduce your reported result of the training from scratch for XSum. However, the decoder_nll loss is always equal to 0, which is quite weird since it's cross-entropy loss.

If I load your pre-trained model, it is not equal to 0.  Do you know the reason?

**Comment by LittlePeaPea** [comment]
Hi, I'm trying to reproduce the result of XSUM from scratch too, using recommended parameters in README. But I cannot reproduce the ROUGE score, which is much lower than the score reported in the paper. Any suggestions? @qiweizhen , thank you!
